dashboard/config/libraries/variableLog.interpreted.js

```diff
@@ -69,3 +69,13 @@ function storeVariableLogforPrevious() {
   previousVarLog = JSON.parse(JSON.stringify(varLog));
 }
 
+// Returns true if the student has a variable label that starts with "_"
+function noBadVariableLabels() {
+  var result = true;
+  Object.keys(varLog).forEach(function (label,index) {
+    if (label.charAt(0) == "_") {
+      result = false;
+    }
```

Comment:
We should use `===` here because it will compare value as well as type. 

Also, not totally sure how `result` is used, but isn't this returning false if the variable starts with `_` ? The comment says it should return true.

Reply:
Good catches, fixed!